apps/paymaster-admin/src/app/api/user-data/route.ts

```diff
@@ -0,0 +1,35 @@
+import { verifyLogInToken } from "@fogo/sessions-sdk";
+import { cacheTag, cacheLife } from "next/cache";
+import type { NextRequest } from "next/server";
+import { NextResponse } from "next/server";
+
+import { connection } from "../../../fogo-connection";
+import { fetchUserPaymasterData } from "../../../server/paymaster";
+
+const fetchData = async (walletAddress: string) => {
+  "use cache";
+  cacheTag("user-data");
+  cacheLife("seconds");
+  return fetchUserPaymasterData(walletAddress);
+};
+
+export const GET = async (request: NextRequest) => {
+  const auth = request.headers.get("authorization");
+  const sessionToken = auth?.startsWith("Bearer ") ? auth.slice(7) : undefined;
+
+  if (!sessionToken)
+    return NextResponse.json({ error: "Missing token" }, { status: 401 });
```

Comment:
Small nit but no reason to use json for errors.  Also please use braces around clauses.